aqueductcore/backend/routers/graphql/resolvers/task_resolver.py

```diff
@@ -30,12 +30,22 @@ async def get_tasks(
             f"Maximum allowed limit for experiments is {MAX_EXPERIMENTS_PER_REQUEST}"
         )
 
+    if filters:
+        experiment = filters.experiment
+        if str(experiment.type) != "IDType.UUID":  # type: ignore
```

Comment:
I think you still allow `experiment` to be `None`, so this is why mypy is complaining on None.type. Probably:

```suggestion
    if filters is not None:
        experiment = filters.experiment
        if experiment is not None and experiment.type != IDType.UUID:
```